src/experiments/generic-experiment/generic-experiment.proto

```diff
@@ -24,7 +24,7 @@ message GenericExperimentMessage {
             TIMEOUT = 8;
             TRAP    = 9;
             WRITE_TEXTSEGMENT = 10;
```

Comment:
Lustige Whitespace-Mischi in dieser Datei. Aber das war schon vorher durchnander ...

Reply:
Ich füg da mal nochmal nen Change dazu, der das vereinheitlicht